squashfs-mount.spec.in

```diff
@@ -0,0 +1,26 @@
+Name:           squashfs-mount
+Version:        SQMNT_VERSION
+Release:        1%{?dist}
+Summary:        setuid mount squashfs utility.
+
+License:        BSD3
+URL:            https://github.com/eth-cscs/squashfs-mount
+Source0:        %{name}-%{version}.tar.gz
+
+BuildArch:  x86_64
```

Comment:
I think this can be dropped, from the docs:

> If not set, the package automatically inherits the Architecture of the machine on which it is built, for example x86_64.

Reply:
Right you are. Fixed.